src/client/pages/Landing/landingPageData.ts

```diff
@@ -25,11 +27,13 @@ type ProductData = {
 
 type ProductsData = Record<Market, ProductData>
 
-export const productsData: ProductsData = {
+export const getProductsData = (): ProductsData => ({
   SE: [
     {
       id: 'swedishNew',
-      linkSlug: '/new',
+      linkSlug: checkFeature(Feature.QUOTE_CART_API)
+        ? '/home-accident-needer'
```

Comment:
Is this the route for the QuouteCart Embark flow? :)

Reply:
Exactly, I followed the same standard as other markets to align SE